src/plugins/privilege/privilege.go

```diff
@@ -218,25 +216,25 @@ func (p *Privilege) loadDBPrivileges(host string, user string) (map[string]dbPri
 				deletePriv: string(r[5].Raw()) == "Y",
 				createPriv: string(r[6].Raw()) == "Y",
 				dropPriv:   string(r[7].Raw()) == "Y",
-				alterPriv:  string(r[8].Raw()) == "Y",
-				indexPriv:  string(r[9].Raw()) == "Y",
+				alterPriv:  string(r[9].Raw()) == "Y",
```

Comment:
OOPS, I forgot `grantpriv` field, it's better to add it here， just not use yet.

Reply:
Have added.